### PR TITLE
feat: More hints to nodes 

### PR DESCRIPTION
**Pull request by michael-radency**
### Added hints to output pane:

**limit reached hint** (if limit and number of returned items equal to this limit)
_"Limit of [limit] items reached. There may be more items that aren't being returned. Tweak the 'Return All' or 'Limit' parameters to access more items."_

**Execute Once hint** (on list like operations 'getAll', 'getMany', 'read', 'search' when input data length more then 1)
_"The operation is performed for each input item. Use the 'Execute Once' setting to only execute it only once for the first input item."_

**expression in field name hint for Set node**
_"An expression is used in 'Fields to Set' in field [someField], did you mean to use it in the value instead?"_

**Split In Batches setup hints**
_"No nodes connected to the 'loop' output of this node"
"The last node in the branch of the 'loop' output must be connected back to the input of this node to loop correctly"_

**HttpRequest node** (if single return item with data property containing array, after execution)
_"The result has a 'data' property which contains an array of items, you can split this array into separate items by using the 'Split Out' node"_

**Aggregate node** (if field to aggregate by not found in any input items, after execution)
_"The field [field] wasn't found in any input item"_

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/NODE-1479/add-more-warnings-to-nodes

### CODE REVIEW

File: packages/editor-ui/src/components/RunData.vue [hunk 1]
```diff
@@ -645,12 +653,100 @@ export default defineComponent({
...
+					if (
+						(this.parentNodeOutputData.length > 1 || this.parentNodePinnedData.length > 1) &&
+						LIST_LIKE_NODE_OPERATIONS.includes((workflowNode.parameters.operation as string) || '')
+					) {
+						const executeOnce = this.workflow.getNode(this.node.name)?.executeOnce;
+
+						if (!executeOnce) {
+							nodeHints.push({
+								message:
+									"The operation is performed for each input item. Use the 'Execute Once' setting to only execute it only once for the first input item.",
```
Comment (ShireenMissi) [inline packages/editor-ui/src/components/RunData.vue:686]:
```suggestion
									"The operation is performed for each input item. Use the 'Execute Once' setting to execute it only once for the first input item.",
```

File: packages/nodes-base/nodes/Transform/SplitOut/SplitOut.node.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import get from 'lodash/get';
 import unset from 'lodash/unset';
+
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Transform/SplitOut/SplitOut.node.ts:3]:
```suggestion
```

File: packages/nodes-base/nodes/Transform/SplitOut/SplitOut.node.ts [hunk 3]
```diff
@@ -147,6 +148,7 @@ export class SplitOut implements INodeType {
 			const multiSplit = fieldsToSplitOut.length > 1;
 
 			const item = { ...items[i].json };
+
```
Comment (ShireenMissi) [inline packages/nodes-base/nodes/Transform/SplitOut/SplitOut.node.ts:151]:
```suggestion
```

**Review by ShireenMissi — COMMENTED** [review]
Love the detailed PR description 🙌
Mostly looks good and I manually tested, all works as expected 🎉
left some minor nitpicks, also is it possible to add some tests?

**Review by ShireenMissi — APPROVED** [review]
LGTM 🎉

**Comment by janober** [comment]
Got released with [`n8n@1.58.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.58.0)

**Pull request merged**